src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -111,7 +138,14 @@ def Check(self, request, context):
                 return _health_pb2.HealthCheckResponse(status=status)
 
     # pylint: disable=arguments-differ
-    def Watch(self, request, context, send_response_callback=None):
+    def Watch(
+            self,
+            request: _health_pb2.HealthCheckRequest,
+            context: grpc.ServicerContext,
+            send_response_callback: Optional[
+                Callable[[_health_pb2.HealthCheckResponse], None]
+            ] = None,
+        ) -> Optional[_Watcher]:        
```

Comment:
`_Watcher` is a private type (indicated by the leading underscore). We cannot make a private type part of our public API. We should either change the name (to Watcher) or we should pull out an interface parent class and maket that the return type of this method.